Improve Section 2 (#77)
gammapy/gammapy-v1.0-paper: src/ms-review.tex

shared across the field. They also frequently imply joint analysis of multi-instrument data.
To extract physically relevant information, the measured counts on an instrument are usually
compared to a parametrised model of the true intensity distribution of \gammaray source in a given

> field-of-view, which is "forward folded" through a model of the instrument response

Reviewer: Then eventually - if you want - you can explain that the dependence on the variables $p_{\rm true}, E_{\rm true}$ in the analytical flux model is typically factorised in a spatial part that usually represents a simple shape (point-source, disk, etc...) and a spectral part that is usually modelled with different types of power law (straight, curved, broken etc..).

And maybe write down
$$\Phi(p_{\rm true}, E_{\rm true}) = \Phi_{\rm spatial}(p_{\rm true})  \Phi_{\rm spectral}(E_{\rm true})$$

> field-of-view, which is "forward folded" through a model of the instrument response

Author: This is now included

  i.e. $E_{\rm disp}(\mu|p_{\rm true}, E_{\rm true})$.
\end{itemize}

> \gammaray data at the Data Level 3 (DL3) therefore consist in lists of gamma-like events and their

Reviewer 2: ```suggestion
 consist in
```
consist of

> \gammaray data at the Data Level 3 (DL3) therefore consist in lists of gamma-like events and their

Reviewer 2: ```suggestion
\gammaray data at the Data Level 3 (DL3) therefore consist of lists of gamma-like events and their
```

Author: Done

corresponding instrument response functions (IRFs). The latter include the effective area,

> point spread function (PSF) and energy dispersion.

Reviewer 2: mention A_eff is effective area and E_disp is energy dispersion?

An additional component of DL3 IRFs is the residual hadronic background model $Bkg$.

> It represents the  intensity of mis-identified charged particles events that are expected

Reviewer: ```suggestion
It represents the intensity of charged particles misidentified as gamma rays that are expected
```

during an observation. It is defined as a function of the measured position in the field-of-view
and measured energy.

In total, the expected number of events in a \gammaray observation is given by:
\begin{align}

> N(p, E) {\rm d}p {\rm d}E =  E_{\rm disp} \star (PSF \star A_{\rm eff} t_{\rm obs} \Phi) + Bkg(p, E)

Author: Simply because the product is done before the convolution

\end{align}

In practice, the background model is subject to numerous uncertainties and needs to be corrected on the data.

> Frequently, a background model is not even available and the background contribution has to be estimated

Author: Done

The next step of the analysis is the data reduction, where all observation events and instrument
responses are filled into or projected onto a common physical coordinate system, defined by
a map geometry. The definition of the map geometry typically consists of a spectral dimension
defined by a binned energy axis and of spatial dimensions, which either define 
a spherical projection from celestial coordinates to a pixelised image space
(for 3-dimensional or cube analysis) or a region on the sky (for regular spectral analysis).
The \code{gammapy.maps} subpackage provides general multidimensional geometry objects
(\code{Geom}) and the associated data structures (\code{Maps}) (see Section~\ref{ssec:gammapy-maps}).

> After all data has been projected into the same geometry, it is typically

Reviewer 2: The necessity for correcting the background on the data has already been mentioned in the last para of the prev subsection. Do you want to put it twice to emphasise the importance, or perhaps it can be merged into one place?
